src/device/sam/atsamd51x-bitfields.go

```diff
@@ -0,0 +1,63 @@
+// Hand created file. DO NOT DELETE.
+// atsamd51x bitfield definitions that are not auto-generated by gen-device-svd.go
+
+// +build sam,atsamd51
+
+// These are the supported pchctrl function numberings on the atsamd51x
+// See http://ww1.microchip.com/downloads/en/DeviceDoc/SAM_D5xE5x_Family_Data_Sheet_DS60001507F.pdf
+// table 14-9
+
+package sam
+
+const (
+	PCHCTRL_GCLK_OSCCTRL_DFLL48     = 0 // DFLL48 input clock source
+	PCHCTRL_GCLK_OSCCTRL_FDPLL0     = 1 // Reference clock for FDPLL0
+	PCHCTRL_GCLK_OSCCTRL_FDPLL1     = 2 // Reference clock for FDPLL1
+	PCHCTRL_GCLK_OSCCTRL_FDPLL0_32K = 3
+	PCHCTRL_GCLK_OSCCTRL_FDPLL1_32K = 3
+	PCHCTRL_GCLK_SDHC0_SLOW         = 3
+	PCHCTRL_GCLK_SDHC1_SLOW         = 3
+	// GCLK_SERCOM[0..7]_SLOW = 3
```

Comment:
It could be named something like GCLK_SERCOM0_SERCOM0_7_SLOW, since it is for a SERCOM common slow clock configuration.

Reply:
Added `PCHCTRL_GCLK_SERCOMX_SLOW` for this.